segments/tokenizer.py

```diff
@@ -130,7 +142,7 @@ def _init_profile(self):
             if grapheme not in self.op_graphemes:
                 self.op_graphemes[grapheme] = 1
             else:
-                raise Exception("You have a duplicate in your orthography profile.")
+                raise Exception("{0} is duplicate in your orthography profile.".format(grapheme))
```

Comment:
Yep, I agree, now that I saw the problems in lingpy. One could also discuss whether one wants to actually throw an error there, as one could also ignore it and just issue a warning. But there needs to be some verbosity of to which characters cause the problem, as it is incredibly problematic to not know which character is causing the problem. Having something like a class-attribute that stores duplicates, and a warning that is thrown if this is encountered: would that be a good pragmatic solution?